### ISSUE TITLE
Cannot fetch dependency from https://registry.npmjs.org blocked by CORS policy

### ISSUE THREAD

**Issue by tchardin** [codesandbox/codesandbox-client#6411]
# 🐛 bug report

## Preflight Checklist

- [x] I have read the [Contributing Guidelines][contributing] for this project.
- [x] I agree to follow the [Code of Conduct][code_of_conduct] that this project
      adheres to.
- [x] I have searched the issue tracker for an issue that matches the one I want
      to file, without success.

## Description of the problem

My sandbox fails to load dependencies and displays the overlay screen with error: 
> Could not fetch dependencies, please try again in a couple seconds: Failed to fetch

In the browser console I can see the more detailed error:
> jfr1k.csb.app/:1 Access to fetch at 'https://registry.npmjs.org/@achingbrain/node-fetch/-/node-fetch-2.6.7.tgz' from origin 'https://jfr1k.csb.app' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource. If an opaque response serves your needs, set the request's mode to 'no-cors' to fetch the resource with CORS disabled.

## How has this issue affected you? What are you trying to accomplish?

While deploying a sandbox template with codesandbox CI from my library repo. The sandbox generated cannot load dependencies. The aim is to view a generated sandbox for new open PR.

### To Reproduce

Either look at the generated [sandbox](https://codesandbox.io/s/myelnet-myel-js-jfr1k) or generate a new one by opening a PR on the [repo](https://github.com/myelnet/myel.js).

### Your Environment

| Software         | Name/Version |
| ---------------- | ------------ |
| Сodesandbox      |       262b71199       |
| Browser          |       Chrome Version 96.0.4664.55 (Official Build) (x86_64)       |
| Operating System |      macOS Catalina 10.15.6        |

[code_of_conduct]: https://github.com/codesandbox/codesandbox-client/blob/master/CODE_OF_CONDUCT.md
[contributing]: https://github.com/codesandbox/codesandbox-client/blob/master/CONTRIBUTING.md

**Comment by JamesACS** [comment]
Hey @tchardin - Thanks for flagging this.

There is a dependency in the dependency graph that defines 'https://registry.npmjs.org/@achingbrain/node-fetch/-/node-fetch-2.6.7.tgz' as a dependency. Our bundler tries to download from that URL, but it fails because registry.npmjs.com doesn't have a CORS policy.

We've opened a PR with a fix here: https://github.com/codesandbox/codesandbox-client/pull/6421

**Comment by djejaquino** [comment]
Hi, I am facing a similar issue. @JamesACS can you please advise if I should open another ticket?

![Screen Shot 2022-01-26 at 10 20 29](https://user-images.githubusercontent.com/5549408/151170280-91d2ed9e-587a-415b-962c-8dfd720ece24.png)

**Comment by tchardin** [comment]
Thanks! Confirming this fixed my issue. @djejaquino yours doesn't seem like a CORS issue, might need a separate one.

**Comment by djejaquino** [comment]
Looks like it is Brave Browser blocking that specific dependency request. I'll dig further and open a ticket in their community hub if confirmed.
